### ISSUE TITLE
No events in fresh demo instance

### ISSUE THREAD

**Issue by mariusandra** [PostHog/posthog#3283]
## Bug description

1. `dropdb posthog` && new dev setup
2. Go through preflight/quiz and get dropped into a demo environment that says "Explore everything in this full-fledged demo environment. "
3. See no events:
![image](https://user-images.githubusercontent.com/53387/107480474-f1b1a500-6b7c-11eb-83f0-08ffaea5eff3.png)

To be fair, the events are there, so I can do this dance:

![2021-02-10 08 50 33](https://user-images.githubusercontent.com/53387/107480608-26256100-6b7d-11eb-8f53-cbd2c6277b3c.gif)

## Environment

- [ ] PostHog Cloud
- [z] self-hosted PostHog, version/commit: _please provide_

## Additional context

It's caching obviously, but looks like a degraded experience.

I would also really love to have an ever persistent "refresh" button next to insights, so that when anything in my data changes I can hit it and see the changes, instead of being persistently stuck in the loop of: 1) reloading the page, 2) realising it's still cached, 3) tweaking the filter to get fresh data

**Comment by macobo** [comment]
To clarify: the issue lies in not dropping the redis cache and the previously cached result being returned rather than with demo data creation?

**Comment by mariusandra** [comment]
Yes, demo data creation itself seems fine. However from an end user standpoint, there's no demo data unless you know to either click "events" or muck with the filters.

**Comment by Twixes** [comment]
I believe we no longer offer this in-app demo option.
